TMTumblrSDK/AppClient/TMTumblrAppClient.h

```diff
@@ -21,10 +21,16 @@
 /// View the authenticated user's dashboard
 + (void)viewDashboard;
 
+/// View the explore page
++ (void)viewExplore;
+
+// View the activity page for a given blog. If blogName is nil, will display the primary blog.
++ (void)viewActivity:(NSString *)blogName;
+
 /// View a tag
 + (void)viewTag:(NSString *)tag;
 
-/// View a blog
+/// View a blog. If blogName is nil, will display the primary blog
 + (void)viewBlog:(NSString *)blogName;
```

Comment:
@whydna Think we should add a `viewPrimaryBlog` method that just calls through with `nil`?